### PR TITLE
Nightly Pipeline

### PR DESCRIPTION
**Pull request by jacksolirius**
### Change description ###
Code to set up the ET Nightly Pipeline 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/gatling/simulations/simulations/ET_Simulation.scala [hunk 1]
```diff
@@ -66,16 +80,47 @@ class ET_Simulation extends Simulation {
         session
     }
 
+  //defines the Gatling simulation model, based on the inputs
+  def simulationProfile(simulationType: String, numberOfPerformanceTestUsers: Double, numberOfPipelineUsers: Double): Seq[OpenInjectionStep] = {
+    simulationType match {
+      case "perftest" =>
+        if (debugMode == "off") {
+          Seq(
+            rampUsers(numberOfPerformanceTestUsers.toInt) during (testDurationMins minutes)
```
Comment (jonsoloway-hmcts) [inline src/gatling/simulations/simulations/ET_Simulation.scala:89]: Is there a reason you've moved away from the open model simulation type here? We usually don't use rampUsers unless it's specifically needed for the test type.

File: src/gatling/simulations/simulations/ET_Simulation.scala [hunk 2]
```diff
@@ -66,16 +80,47 @@ class ET_Simulation extends Simulation {
...
+          )
+        }
+        else {
+          Seq(atOnceUsers(1))
+        }
+      case "pipeline" =>
+        Seq(rampUsers(numberOfPipelineUsers.toInt) during (2 minutes))
```
Comment (jonsoloway-hmcts) [inline src/gatling/simulations/simulations/ET_Simulation.scala:96]: This should now have a 'dot' in, from 3.9.x => 2.minutes

File: src/gatling/simulations/simulations/ET_Simulation.scala [hunk 3]
```diff
@@ -48,14 +62,14 @@ class ET_Simulation extends Simulation {
...
-    //  .repeat(1){
+      //  .repeat(1){
       exec(  _.set("env", s"${env}"))
-      .exec(flushHttpCache)
-      .exec(flushCookieJar)
-      .feed(UserFeederET)
+        .exec(flushHttpCache)
+        .exec(flushCookieJar)
+        .feed(UserFeederET)
         .repeat(1) {
```
Comment (jonsoloway-hmcts) [inline src/gatling/simulations/simulations/ET_Simulation.scala:70]: Is this loop necessary?

**Review by jonsoloway-hmcts — COMMENTED** [review]
A few minor comments for review.

File: src/gatling/simulations/simulations/ET_Simulation.scala (hunk 3 above)
Comment (jacksolirius) [reply src/gatling/simulations/simulations/ET_Simulation.scala:70]: Not really, was used during the script making process. I have taken it out

File: src/gatling/simulations/simulations/ET_Simulation.scala (hunk 2 above)
Comment (jacksolirius) [reply src/gatling/simulations/simulations/ET_Simulation.scala:96]: Changed

**Review by jonsoloway-hmcts — APPROVED** [review]
👍

**Pull request merged**